Image refresh misc batch (#7725)
fishtown-analytics/docs.getdbt.com: website/docs/guides/custom-cicd-pipelines.md

When running a CI/CD pipeline you’ll want to use a service token instead of any individual’s API key. There are [detailed docs](/docs/dbt-cloud-apis/service-tokens) available on this, but below is a quick rundown (this must be performed by an Account Admin):

> - Login to your <Constant name="cloud" /> account.

Reviewer: ```suggestion
1. Log in to your <Constant name="cloud" /> account.
```

> - In the upper left, click the menu button, then **Account settings**.

Reviewer: ```suggestion
2. Click your account name at the bottom left-hand menu and go to **Account settings**.
```

> - Click [**Service tokens**](/docs/dbt-cloud-apis/service-tokens) on the left.

Reviewer: ```suggestion
3. Click [**Service tokens**](/docs/dbt-cloud-apis/service-tokens) on the left.
```

> - Click **+ Create service token** to create a new token specifically for CI/CD API calls.

Reviewer: ```suggestion
4. Click **+ Create service token** to create a new token specifically for CI/CD API calls.
```

> - Name your token something like “CICD Token”.

Reviewer: ```suggestion
5. Name your token something like “CICD Token”.
```

> - Click the **+Add permission** button under **Access**, and grant this token the **Job Admin** permission.

Reviewer: ```suggestion
6. Click the **+Add permission** button under **Access**, and grant this token the **Job Admin** permission.
```

> - Click **Save** and you’ll see a grey box appear with your token. Copy that and save it somewhere safe (this is a password, and should be treated as such).

Reviewer: ```suggestion
7. Click **Save** and you’ll see a grey box appear with your token. Copy that and save it somewhere safe (this is a password, and should be treated as such).
```
